ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-master.yaml

```diff
@@ -27,6 +27,8 @@ images:
       paths:
       - destination_dir: .
         source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build/_output/bin/performance-addon-operators
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift-kni/performance-addon-operators/build/_output/bin/performance-profile-creator
```

Comment:
I was too hasty with my lgtm. It's very possivle we will need a new `input` definition. Need to check.